tools/wpt/browser.py

```diff
@@ -315,6 +322,33 @@ def install_webdriver(self, dest=None):
             untar(get(url).raw, dest=dest)
         return find_executable(os.path.join(dest, "geckodriver"))
 
+    def install_geckodriver_nightly(self, dest):
+        import tarfile
+        import mozdownload
```

Comment:
If you are using mozdownload here, where is the update for a requirements.txt, or setup.py file?

Reply:
It's already used to download Firefox.

Reply:
Oh, cool! Didn't know that. Good that there is another customer of the project I own, and that our changes are still useful.

With all that the changes look fine to me. But would be good to get a final review from a person who has more knowledge on wpt than me.